app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -158,6 +159,11 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = ""
     )
 
+    var currentWallpaper by stringPreference(
```

Comment:
Can we add a kDoc for this preference

Reply:
> A comment is a failure to express yourself in code.  If you fail, then write a comment; but try not to fail. <Uncle Bob Martin>
XD
https://twitter.com/unclebobmartin/status/870311898545258497

Reply:
When something is explicit we don't need docs :)